GuidelinesForNewComponents.md

```diff
@@ -0,0 +1,28 @@
+# Guidelines for Contributing New Components to Open Data Hub
+
+This document describes the guidelines to submit new ideas such as new features and components to Open Data Hub. This specifically pertains to new components or large features that require an architecture review or a re-design of existing components. This documents does not apply to small or incremental features, for those please open an issue with a "feature" label in the [Open Data Hub project](https://issues.redhat.com/projects/ODH/issues/ODH-380?filter=allopenissues)
```

Comment:
You can remove `ODH-380` from the url. It looks like Jira will automatically redirect to the first open issue with this filter enabled

Reply:
Also there is a typo: `This documents does not` -> `This document does not`